lib/engine/game/g_1848/step/tasmania_tile.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
```

Comment:
why do you need a special step for tasmania tile?

Reply:
Special case in the rule book.
If the privates close and the Tasmania tile hasn't placed (an ability of one of the privates) the owning player or corp must place it immediately. This step is to handle this scenario.

Reply:
Okay